atd-vze/src/Components/GridExportData.js

```diff
@@ -98,7 +98,31 @@ const GridExportData = ({ query, columnsToExport, totalRecords }) => {
       return flattenedRow;
     });
 
-    return flattenedData;
+    const cleanedAndFlattenedData = flattenedData.map(item => {
+      // Some fields are presenting issues when they are exported as .csv and
+      // imported into MSFT Excel, Acces & Power BI. For now, we're just going to
+      // modify the data for export to solve issues related to returns and double-quotes
+      // In the future, we may choose to clean these values at the database level.
+
+      const columnsToClean = [
```

Comment:
Good call on specifying which fields to avoid unnecessary cleanup and make this faster.